### PR TITLE
refactor: Converted ValidateUtil JavaClass to Kotlin Object

### LINKED ISSUE #903: chore: Convert ValidateUtil class to Kotlin Object
**Summary:**
Currently, the ValidateUtil class is written in java, it should be translated to kotlin instead which makes the code much more legible and cleaner.

**Steps to reproduce:**
Since it is a chore issue so it can directly be seen in the codebase at 
`mobile-wallet/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils`

**Expected behavior:**
It should be written in kotlin instead

**Observed behavior:**
The concerned class is written in java

**Device and Android version:**
NA

**Screenshots:**
NA

### PR DESCRIPTION
**Pull request by s-ayush2903**
## Issue Fix
Fix #903 

## Screenshots
NA

## Description
Upgraded the KotlinVersion and converted the concerned java class to kotlin Object

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,7 +35,7 @@ ext {
     targetSdkVersion = 28
     compileSdkVersion = 28
     buildToolsVersion = '26.0.2'
-    kotlinVersion = "1.3.21"
+    kotlinVersion = "1.3.61"
```
Comment (s-ayush2903) [inline build.gradle:38]: Updated as the previous one was quite outdated and I was not fully sure about all the features of kt version 1.3.71 .

**Comment by s-ayush2903** [comment]
Do i need to rebase those two commits on top of the first one? If I do so the, it'll be showing more file changes which ain't in the context of the issue this PR targets
Please review.

**Comment by jawidMuhammadi** [comment]
Squash your commits into a single one, please.

**Comment by s-ayush2903** [comment]
Please Review

File: build.gradle [hunk 2]
```diff
@@ -11,8 +12,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:3.1.3'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
         classpath 'com.google.gms:google-services:4.0.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21"
-
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```
Comment (devansh-299) [inline build.gradle:15]: @s-ayush2903 directly specify the kotlin version, no need to define a property.

**Comment by rachittt** [comment]
Hello @s-ayush2903, Can you edit your commit title and use "refactor:" instead of "chore:"

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/ValidateUtil.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package org.mifos.mobilewallet.mifospay.utils
+
+import android.text.TextUtils
+import android.util.Patterns
+
+object ValidateUtil {
+  fun isValidEmail(email: String?): Boolean {
+    return (!TextUtils.isEmpty(email)
```
Comment (rachittt) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/ValidateUtil.kt:8]: As far as I know, you can convert a function into compact and omit the return keyword. What do you think?

**Comment by s-ayush2903** [comment]
I apologize, it didn't click at that moment

**Pull request merged**